frame/instrumental-strategy-pablo/src/lib.rs

```diff
@@ -246,12 +242,7 @@ pub mod pallet {
 			asset_id: T::AssetId,
 			pool_id: T::PoolId,
 		) -> DispatchResultWithPostInfo {
-			let who = ensure_signed(origin)?;
-			let access_right = Self::admin_accounts(who).ok_or(Error::<T>::NotAdminAccount)?;
-			ensure!(
-				access_right == AccessRights::Full || access_right == AccessRights::SetPoolId,
-				Error::<T>::NotEnoughAccessRights
-			);
+			T::ExternalOrigin::ensure_origin(origin)?;
```

Comment:
GitHub doesn't allow to create comments for code that has not been changed (strange behavior I think). I think these errors are no longer needed:
- `NotEnoughAccessRights`
- `NotAdminAccount`

Reply:
Sure!

Reply:
Sure!